### DISCUSSION TITLE
When a release with not javax?

### DISCUSSION THREAD

**Discussion by rayacode** [playframework/playframework#12919] [General]
Hi everyone thank you a lot for contributing and maintaining the play framework. i currently use snapshot localbuild because of hibernate and javax problems, can i ask for a faster release to a 3.1.0-snapshot? also i found this snapshot has problems in cleaning files that have been changed or deleted mostly js won't effect in dist until a complete clean and rebuild.

**Comment by mkurz** [comment]
> can i ask for a faster release to a 3.1.1-snapshot?

what do you mean? releasing `3.1.0-SNAPSHOT` to maven central?

> also i found this snapshot has problems in cleaning files that have been changed or deleted mostly js won't effect in dist until a complete clean and rebuild.

Can you please describe a bit more detailed what you problem is?

**Comment by rayacode** [comment]
my problem is current playframework version 3.0.5 still uses javax, and needs javax validation package, i use localbuild 3.1.0-snapshot, a build with current state of playframework source, i'm asking to faster release to a version that no longer have problems with javax, and updated to the jakarta imports, and usage of hibernate 7.

**Reply by ihostage** [reply]
Every commit in `main` branch Play publishes artifacts to [Sonatype Snapshots repository](https://oss.sonatype.org/content/repositories/snapshots/org/playframework/). You can use it instead of local build 😉 
For example https://github.com/playframework/play-samples/blob/1400ae92d17f18e11211917ee836e72b22afa363/java/jpa/project/plugins.sbt#L2-L3

**Reply by rayacode** [reply]
@ihostage  thank you!
